example/Supfile

```diff
@@ -29,6 +29,8 @@ networks: # Groups of hosts
       - ubuntu@stg.example.com
 
   prod:
+    inventory:
+      - for i in 4 5 6; do echo "$USER@prod$i.example.com\n"; done
```

Comment:
I kind of like the word inventory (similar to `host_inventory` in the original proposal - https://github.com/pressly/sup/commit/6326b57447a1fa34bccf09b65663ec03ea43fb12), but I'd rather be more specific that user is supposed to provide "command" to be run.

```yaml
networks:
  stg:
    hosts:
      - stg1.example.com
      - stg2.example.com
  prod:
    hosts_run: for i in 4 5 6; do echo "$USER@prod$i.example.com\n"; done
``` 

My suggestion would be using `hosts_run` or `hosts_from` instead of `inventory`. What do you guys think?